client/src/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { App } from './App.jsx';
+
+const appElement = document.getElementById('app');
+
+ReactDOM.render(<App />, appElement);
```

Comment:
React recently made some updates. Have you researched how to [render](https://reactjs.org/docs/react-dom-client.html) the app with version 18? (you probably got a message in your devtools console)